app/pandas.ipynb

```diff
@@ -208,16 +275,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
+   "execution_count": 77,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "22\n"
+     ]
+    }
+   ],
    "source": [
-    "# Write your code here"
+    "average_age = int(df[\"Age\"].mean())\n",
+    "print(average_age)"
```

Comment:
it should be new column, not printed value

Reply:
fixed